perl/lib/Sanger/CGP/Vaf/Data/AbstractVcf.pm

```diff
@@ -118,6 +156,15 @@ sub getBedIntervals {
     return shift->{'_b'};
 }
 
+sub _check_file_exists {
+  my ($self, $file) = @_;
+  die "set_input requires a value for $file" unless(defined $file);
```

Comment:
Is this error message appropriate? What is `set_input`?

Reply:
created separate subs to check vcf and bam cram file presence, updated text accordingly